regression/cbmc/union10/union_list2.desc

```diff
@@ -1,4 +1,4 @@
-CORE
+CORE broken-z3-smt-backend
```

Comment:
Why does this break?

Reply:
These appear to have been broken before, previous version of `cbmc` have errors here.